keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/Health.kt

```diff
@@ -17,12 +17,18 @@
  */
 package com.netflix.spinnaker.keel.api.ec2
 
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonInclude.Include.NON_EMPTY
 import java.time.Duration
 
+@JsonInclude(NON_EMPTY)
```

Comment:
Seems like we're putting this in a lot of places. I wonder if we can default to it on the `ObjectMapper`?

Reply:
Great point.. I certainly want this on all collection properties for export. Can you think of any cases where we wouldn't want this enabled by default? If not, I'll clean up these in another PR (since I sprinkled some around previously too.)